lib/management/accesskey.ts

```diff
@@ -29,11 +29,12 @@ const withAccessKey = (sdk: CoreSdk, managementKey?: string) => ({
     keyTenants?: AssociatedTenant[],
     userId?: string,
     customClaims?: Record<string, any>,
+    description?: string,
```

Comment:
please add `description` to the documentation of the method (after line 22), like you did for update